src/testers.jl

```diff
@@ -144,21 +147,22 @@ end
 
 # Arguments
 - `config`: defaults to `ChainRulesTestUtils.ADviaRuleConfig`.
-- `f`: Function to which rule should be applied. Can also provide `f ⊢ f̄`.
-- `args` either the primal args `x`, or primals and their tangents: `x ⊢ x̄`
-    - `x`: input at which to evaluate `f` (should generally be set to an arbitary point in the domain).
-    - `x̄`: currently accumulated cotangent, will be generated automatically if not provided
+- `f`: function for which the `rrule` should be tested. Its tangent can be provided using `f ⊢ f̄`.
+  (You can enter `⊢` via `\\vdash` + tab in the Julia REPL and supporting editors.)
+- `args...`: either the primal args `x`, or primals and their tangents: `x ⊢ x̄`
```

Comment:
```suggestion
- `args...`: either the primal args `x`, or primals and their cotangents: `x ⊢ x̄`
```
? (for consistency with https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/216/files#diff-988b853ad3c22aebac9421d3a31e60f9330789df43bccfb9fb75365048398fbaR154)